src/main/java/graphql/execution/ExecutionStrategyParameters.java

```diff
@@ -187,9 +218,6 @@ public Builder deferredCallContext(DeferredCallContext deferredCallContext) {
         }
 
         public ExecutionStrategyParameters build() {
-            if (deferredCallContext == null) {
-                deferredCallContext = new DeferredCallContext();
```

Comment:
There was no point in creating a new instance of `DeferredCallContext` by default.
With the current change, the presence of an instance of `DeferredCallContext` signifies that the execution is part of a deferred call.